MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/Grouping/IMvxGroupableRecyclerAdapter.cs

```diff
@@ -0,0 +1,9 @@
+using System.Windows.Input;
+
+namespace MvvmCross.Droid.Support.V7.RecyclerView.Grouping
+{
+    public interface IMvxGroupableRecyclerAdapter
+    {
+        ICommand GroupHeaderClickCommand { get; set; }
```

Comment:
These should be `ICommand`.  Mvx isn't the only `ICommand` implementation people use.